packages/ui/src/components/theme-selector-v2/theme-dialog.tsx

```diff
@@ -156,14 +151,38 @@ const ThemeDialog: FC<ThemeDialogProps> = ({
             </Select.Root>
           </div>
 
+          <Separator className="bg-borders-4 h-px" />
+
+          {/* Inset Adjustment */}
+          <div className="grid grid-cols-[246px_1fr] gap-x-8">
```

Comment:
any way to avoid magic numbers?

Reply:
I think the classname speaks for itself: there are two columns, one is 246px, and another takes the rest of the space. Easy to read, no logic is connected to this magic number. And this approach is approach is used widely in the codebase.
I could move it to a separate classname into `style.css` using `@apply` but it doesn't seem reusable.